testproto/unsafe/unsafe_test.go

```diff
@@ -94,7 +100,7 @@ func Test_UnmarshalVTUnsafe(t *testing.T) {
 		assertBytesAreOriginal(t, (t2Unsafe.Sub).(*UnsafeTest_Sub2_).Sub2.B[i], true, originalData2)
 	}
 
-	// map field
+	// map[string]byte field
```

Comment:
nit: `map[string]bytes` might be more accurate

Reply:
good catch, fixed and squashed into https://github.com/planetscale/vtprotobuf/pull/127/commits/c47c2bc1dff9ed28b324d5519c7f1a31cae0309c